src/App.tsx

```diff
@@ -1,34 +1,12 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
Add the rule for @typescript-eslint/no-shadow into your eslint.json's rules rather than on that specific file